### PR TITLE
Update physical kit mentions of Python version to 3.10

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This doesn't touch the simulator, which will need a separate release to make this change.

I've skimmed through all the code examples and didn't spot anything which needed updating.

Fixes https://github.com/srobo/docs/issues/346

### CODE REVIEW

File: programming/python/index.md [hunk 1]
```diff
@@ -15,11 +15,11 @@ There are a number of tutorials out there which might help you to learn to progr
...
-* [The Official Tutorial](https://docs.python.org/3.9/tutorial/) -- the good stuff starts at chapter 3,
+* [The Official Tutorial](https://docs.python.org/3.10/tutorial/) -- the good stuff starts at chapter 3,
     but you should at least skim the stuff before it;
     there is a lot there and it may be a little overwhelming.
 
 * A number of tutorials for beginners are linked to from [here](https://wiki.python.org/moin/BeginnersGuide/NonProgrammers).
     [Wikibooks Tutorial](https://en.wikibooks.org/wiki/Non-Programmer%27s_Tutorial_for_Python_3) seems like a good one.
 
-* [The Official Docs](https://docs.python.org/3.9/), for the version of python on the [Odroid Brain Boards](/docs/kit/brain_board).
+* [The Official Docs](https://docs.python.org/3.10/), for the version of python on the [Odroid Brain Boards](/docs/kit/brain_board).
```
Comment (antoinepetty) [inline programming/python/index.md:25]: Should we be talking about Pis here now?

Comment (PeterJCLaw) [reply programming/python/index.md:25]: Yeah, I just noticed that too. We should, though I think there's a separate task for that (it involves a deeper change to the brain board docs). Happy to hold this change and let it be part of that PR though.

**Review by antoinepetty — APPROVED** [review]

**Review by raccube — APPROVED** [review]
Excluding the odroid thing (already addressed in @antoinepetty's review) looks good.

File: programming/python/index.md (hunk 1 above)
Comment (trickeydan) [reply programming/python/index.md:25]: Let's hold this change until later. 👍

Comment (PeterJCLaw) [reply programming/python/index.md]: rebased on #355.

**Comment by trickeydan** [comment]
Hang on what on earth has happened here with the commits?

**Comment by trickeydan** [comment]
I've rebased onto `origin/dgt/brain-odroid-to-raspberry-pi`. I think GitHub just struggles to understand what the history was doing.

**Comment by PeterJCLaw** [comment]
Ah, I think the issue is that I rebased this onto an earlier version of #355, before you TinyPNG'd it. I'll fix.

**Pull request merged**